migrate.txt

```diff
@@ -0,0 +1,276 @@
+--
```

Comment:
The usual extension for these kinds of files is `.psql`.

Reply:
done